### PR TITLE
Code cleanup due to boredom in planning

### PR DESCRIPTION
**Pull request by johnoneill46**
- Updated ReadMe to be more clearer (Added proper reference to License)
- Removed millions of redundant empty lines
- Reformatted to make code more readable
- Made access modifiers more strict - e.g. public -> protected/private etc
- Optimised imports
- Removed redundant "throws exception" from methods which don't throw exceptions
- Replaced "Arrays.asList()" with "Collections.singletonList()" for cases where only a single element was ever being added

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
     </suppress>
-
     <suppress>
-        <notes>Vulnerability applies to Netty before version 4.1.42, project does not use Netty</notes>
+        <notes>Vulnerability applies to Netty before version 4.1.44, project does not use Netty</notes>
         <cve>CVE-2014-3488</cve>
         <cve>CVE-2015-2156</cve>
         <cve>CVE-2019-16869</cve>
+        <cve>CVE-2019-20444</cve>
+        <cve>CVE-2019-20445</cve>
```
Comment (johnoneill46) [inline dependency-check-suppressions.xml]: These suppressions were necessary as there a new vulnerabilities in Netty which is used in the java-application-insights dependency unfortunately

File: src/test/java/uk/gov/hmcts/reform/emclient/service/EvidenceManagementDeleteServiceImplTest.java [hunk 2]
```diff
@@ -45,12 +45,6 @@ public void setUp() {
         when(userService.getUserDetails(anyString())).thenReturn(UserDetails.builder().id("19").build());
     }
 
-    /**
-     * This test issues a document delete request that is expected to succeed. It ensures that the OK response from
-     * the EM document store service passes cleanly through the evidence management client api to the caller
-     * without any issues or exceptions occurring.
-     * <p/>
-     */
```
Comment (jakub-kozlowski) [inline src/test/java/uk/gov/hmcts/reform/emclient/service/EvidenceManagementDeleteServiceImplTest.java]: why removing test descriptions? I agree test name should ideally say what the test is doing, but if the description is already there - it doesn't do any harm?

File: dependency-check-suppressions.xml (hunk 1 above)
Comment (johnoneill46) [reply dependency-check-suppressions.xml]: Yep you're right - was able to remove all suppressions now for Netty 👍

File: src/test/java/uk/gov/hmcts/reform/emclient/service/EvidenceManagementDeleteServiceImplTest.java (hunk 2 above)
Comment (johnoneill46) [reply src/test/java/uk/gov/hmcts/reform/emclient/service/EvidenceManagementDeleteServiceImplTest.java]: Reintroduced 👍

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 HMCTS (HM Courts & Tribunals Service)
+Copyright (c) 2020 HMCTS (HM Courts & Tribunals Service)
```
Comment (yarpo) [inline LICENSE]: shouldn't b that 2018-2020?

Comment (johnoneill46) [reply LICENSE]: 🤷‍♂  Not 100% sure, in every other repo we just have the single year

File: build.gradle [hunk 4]
```diff
@@ -138,6 +138,7 @@ def versions = [
         reformPropertiesVolume                  : '0.0.4',
         reformsHttpProxySpringBootAutoconfigure : '1.1.0',
         reformsJavaLogging                      : '5.0.1',
```
Comment (yarpo) [inline build.gradle]: do we still use it?

File: dependency-check-suppressions.xml [hunk 5]
```diff
@@ -1,16 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-
     <suppress>
         <notes>Vulnerability applies to Spring Framework 5.0.5, project uses 5.2.0</notes>
         <cve>CVE-2018-1258</cve>
     </suppress>
-
-    <suppress>
```
Comment (yarpo) [inline dependency-check-suppressions.xml]: <3

**Pull request merged**